src/ripple/core/impl/Config.cpp

```diff
@@ -439,29 +441,22 @@ void Config::loadFromString (std::string const& fileContents)
         validatorsFile = strTemp;
 
         if (validatorsFile.empty ())
-        {
-            JLOG (j_.error()) <<
-                "[" SECTION_VALIDATORS_FILE "]" <<
-                ": " << strTemp <<
-                " is not a valid path";
-            validatorsFile.clear ();
-        }
-        else if (!boost::filesystem::exists (validatorsFile))
-        {
-            JLOG (j_.error()) <<
-                "[" SECTION_VALIDATORS_FILE "]" <<
-                ": the file " << validatorsFile <<
-                " does not exist";
-            validatorsFile.clear ();
-        }
-        else if (!boost::filesystem::is_regular_file (validatorsFile))
-        {
-            JLOG (j_.error()) <<
-                "[" SECTION_VALIDATORS_FILE "]" <<
-                ": the file " << validatorsFile <<
-                " is not a regular file";
-            validatorsFile.clear ();
-        }
+            Throw<std::runtime_error> (
+                "Invalid path specified in [" SECTION_VALIDATORS_FILE "]");
+
+        if (!validatorsFile.is_absolute())
+            validatorsFile = CONFIG_DIR / validatorsFileName;
```

Comment:
Here's the first reason that Appveyor is failing. `validatorsFileName` is the default. This should be
```validatorsFile = CONFIG_DIR / validatorsFile;```